src/NLog/GlobalDiagnosticsContext.cs

```diff
@@ -57,22 +57,42 @@ public static void Set(string item, string value)
         }
 
         /// <summary>
+        /// Sets the Global Diagnostics Context item to the specified value.
+        /// </summary>
+        /// <param name="item">Item name.</param>
+        /// <param name="value">Item value.</param>
+        public static void Set(string item, object value)
+        {
+            lock (dict)
+            {
+                dict[item] = value;
+            }
+        }
+
+        /// <summary>
         /// Gets the Global Diagnostics Context named item.
         /// </summary>
         /// <param name="item">Item name.</param>
         /// <returns>The item value of string.Empty if the value is not present.</returns>
```

Comment:
Reworded it completely, now reads: <returns>The value of <paramref name="item"/>, if defined; otherwise <see cref="String.Empty"/>.</returns>

I believe there may be a few more of these floating around, I'll check and update them as well.